receiver/jmxreceiver/config.go

```diff
@@ -34,23 +35,17 @@ type config struct {
 	GroovyScript string `mapstructure:"groovy_script"`
 	// The duration in between groovy script invocations and metric exports (10 seconds by default).
 	// Will be converted to milliseconds.
-	Interval time.Duration `mapstructure:"interval"`
+	CollectionInterval time.Duration `mapstructure:"collection_interval"`
 	// The JMX username
 	Username string `mapstructure:"username"`
 	// The JMX password
 	Password string `mapstructure:"password"`
-	// The metric exporter to use ("otlp" or "prometheus", "otlp" by default).
-	Exporter string `mapstructure:"exporter"`
-	// The OTLP Receiver endpoint to send metrics to ("localhost:55680" by default).
+	// The OTLP Receiver endpoint to send metrics to ("0.0.0.0:<random open port>" by default).
 	OTLPEndpoint string `mapstructure:"otlp_endpoint"`
 	// The OTLP exporter timeout (5 seconds by default).  Will be converted to milliseconds.
-	OTLPTimeout time.Duration `mapstructure:"otlp_timeout"`
+	exporterhelper.TimeoutSettings `mapstructure:",squash"`
 	// The headers to include in OTLP metric submission requests.
-	OTLPHeaders map[string]string `mapstructure:"otlp_headers"`
-	// The Prometheus Host
-	PrometheusHost string `mapstructure:"prometheus_host"`
-	// The Prometheus Port
-	PrometheusPort int `mapstructure:"prometheus_port"`
+	Headers map[string]string `mapstructure:"headers"`
```

Comment:
Ok, I am keep pushing for this, but we should keep things consistent, at least have the same names, I want the config to look like this in yaml:

```yaml
otlp:
  # Fields under otlp or otlp_config should match the name of the fields in otlp.Config.
  endpoint: ...
  timeout: ...
  headers: ...
```